docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -3542,173 +3542,312 @@ SELECT toDateTime64('2019-01-01 00:00:00', 3, 'Asia/Istanbul') AS value, toTypeN
 
 ## toDateTime64OrDefault
 
-## toDecimal(32\|64\|128\|256)
+## toDecimal32
 
-Converts `value` to the [Decimal](../data-types/decimal.md) data type with precision of `S`. The `value` can be a number or a string. The `S` (scale) parameter specifies the number of decimal places.
+Converts an input value to a value of type [`Decimal(9, S)`](../data-types/decimal.md) with scale of `S`. Throws an exception in case of an error.
 
-- `toDecimal32(value, S)`
-- `toDecimal64(value, S)`
-- `toDecimal128(value, S)`
-- `toDecimal256(value, S)`
+**Syntax**
 
-## toDecimal(32\|64\|128\|256)OrNull
+```sql
+toDecimal32(expr, S)
+```
 
-Converts an input string to a [Nullable(Decimal(P,S))](../data-types/decimal.md) data type value. This family of functions includes:
+**Arguments**
 
-- `toDecimal32OrNull(expr, S)` — Results in `Nullable(Decimal32(S))` data type.
-- `toDecimal64OrNull(expr, S)` — Results in `Nullable(Decimal64(S))` data type.
-- `toDecimal128OrNull(expr, S)` — Results in `Nullable(Decimal128(S))` data type.
-- `toDecimal256OrNull(expr, S)` — Results in `Nullable(Decimal256(S))` data type.
+- `expr` — Expression returning a number or a string representation of a number. [Expression](../syntax.md/#syntax-expressions).
+- `S` — Scale parameter from [ 1 : 9 ] specifying how many decimal digits the fractional part of a number can have. [UInt8](../data-types/int-uint.md).
 
-These functions should be used instead of `toDecimal*()` functions, if you prefer to get a `NULL` value instead of an exception in the event of an input value parsing error.
+Supported arguments:
```

Comment:
Minor comment and kind of related: In the `to(U)Int` functions you documented already above, we have as supported arguments:

```md
Supported arguments:

- Values of type (U)Int8/16/32/64/128/256.
- Values of type Float32/64.
- String representations of (U)Int8/16/32/128/256.
```

For consistency with l. 3561-3562, we should rewrite that to

```md
Supported arguments:

- Values or string representations of type (U)Int8/16/32/64/128/256.
- Values of type Float32/64.
```